src/libs/ontouml2db/factory/Factory.ts

```diff
@@ -0,0 +1,316 @@
+/**
```

Comment:
We are very puzzled about this solution. Why do you need to recreate the model as a graph to then transform it to RDB? Couldn't you simply use the methods in the ModelManager?

Reply:
The transformation process to DB needs to add specific rules, such as primary and foreign keys.

Another point is regarding the class traceability for the tables, necessary to produce the Ontop .odba file, that is, I need to know in which table the data of each class are stored and how to filter them after the transformation.